gme/gme.cpp

```diff
@@ -181,9 +181,9 @@ gme_err_t gme_open_file( const char* path, Music_Emu** out, int sample_rate )
 		header_size = sizeof header;
 		RETURN_ERR( in.read( header, sizeof header ) );
 		file_type = gme_identify_extension( gme_identify_header( header ) );
+		if ( !file_type )
+			return gme_wrong_file_type;
 	}
-	if ( !file_type )
-		return gme_wrong_file_type;
```

Comment:
Is this not cosmetic? Or am I missing something?

Reply:
No, because if file_type is non-zero, then it's still going to be non-zero when it skips over the if clause.